src/Theme.js

```diff
@@ -1,9 +1,59 @@
-import { createMuiTheme } from '@material-ui/core/styles';
+import { createTheme } from '@codeforafrica/hurumap-ui';
 
 const FONT_FAMILY_HEADING = '"Lora", serif';
 const FONT_FAMILY_TEXT = '"Muli", sans-serif';
 
-const Theme = createMuiTheme({
+const COLOR_BREWER_DIVERGING = [
+  '#a50026',
+  '#d73027',
+  '#f46d43',
+  '#fdae61',
+  '#fee08b',
+  '#ffffbf',
+  '#d9ef8b',
+  '#a6d96a',
+  '#66bd63',
+  '#1a9850',
+  '#006837'
+].reverse();
+
+const Theme = createTheme({
+  chart: {
+    pie: {
+      legendWidth: 50,
```

Comment:
This is kind of small

Reply:
Larger one makes pie charts look massive.
@KhadijaMahanga can edit this once she renders the chart definitions she is working on.